logzio-sender/src/main/java/io/logz/sender/LogzioSender.java

```diff
@@ -163,7 +173,7 @@ public void drainQueueAndSend() {
 
     public void send(JsonObject jsonMessage) {
         // Return the json, while separating lines with \n
-        logsBuffer.enqueue((jsonMessage+ "\n").getBytes());
+        logsBuffer.enqueue((jsonMessage+ "\n").getBytes(Charset.forName("UTF-8")));
```

Comment:
You are running a lookup (look at the implementation) on each send() - too expensive. Use the constant `StandardCharsets.UTF_8` instead.